main.py

```diff
@@ -99,6 +99,8 @@ async def listen_to_websocket(
                     fill_events = feed_handler.handle(response)
                     if conf['print_fills']:
                         print_fills(fill_events, cpid_to_market_info)
+                    if conf['print_subaccounts']:
+                        print_subaccounts(feed_handler.get_subaccounts())
```

Comment:
this is gonna print the subaccounts on every block regardless of update, can we make it so that it only prints on update?